core-services/image-mirroring/openshift/mapping_origin_4_4

```diff
@@ -174,6 +174,7 @@ registry.svc.ci.openshift.org/origin/4.4:node-problem-detector quay.io/openshift
 registry.svc.ci.openshift.org/origin/4.4:node-problem-detector-operator quay.io/openshift/origin-node-problem-detector-operator:4.4 quay.io/openshift/origin-node-problem-detector-operator:4.4.0
 registry.svc.ci.openshift.org/origin/4.4:oauth-apiserver quay.io/openshift/origin-oauth-apiserver:4.4 quay.io/openshift/origin-oauth-apiserver:4.4.0
 registry.svc.ci.openshift.org/origin/4.4:oauth-proxy quay.io/openshift/origin-oauth-proxy:4.4 quay.io/openshift/origin-oauth-proxy:4.4.0
+registry.svc.ci.openshift.org/origin/4.4:oauth-proxy-samples quay.io/openshift/origin-oauth-proxy-samples:4.4 quay.io/openshift/origin-oauth-proxy-samples:4.4.0
```

Comment:
this ugly name `origin-oauth-proxy-samples` is just for our build process, right? The imagestream in samples operator will be name oauth-proxy again, won't it?

Reply:
@sttts we pick whatever name we want in samples operator image stream (iow. `openshift/oauth-proxy:v4.4` would be my preferred.) We only need to distinguish the regular oauth-proxy image from the one we plan to use in image stream.

Reply:
ok, sounds good.